### DISCUSSION TITLE
Update Wiki (Token revoke is possible)

### DISCUSSION THREAD

**Discussion by tispokes** [homeassistant-apps/app-cloudflared#644] [Ideas]
https://dash.cloudflare.com/profile/api-tokens

**Comment by elcajon** [comment]
Thanks for the info. We referred to this [issue](https://github.com/cloudflare/cloudflared/issues/93) in our wiki. As the issue seems to be resolved I guess we could remove the information from our wiki.
What do you think @brenner-tobias?

**Reply by brenner-tobias** [reply]
Agreed, thanks for rainsing this @tispokes. I just removed the section from the wiki, feel free to let me know if this is documented anywhere else.
